index.html

```diff
@@ -61,7 +61,7 @@
 <body class="bg-img">
 
 
-  <header class="text-center title"><b>CALCULATOR | YOSHITHA</b></header>
+  <header class="text-center title" style="font-family: 'Poppins', sans-serif; font-size: 3.5rem;"><b>CALCULATOR | YOSHITHA</b></header>
```

Comment:
Don't use internel/inline css use the styles.css file to add custom css @Yushmanth-reddy